components/rmrk/Gallery/CollectionPriceChart.vue

```diff
@@ -1,160 +1,180 @@
+                x: data[0],
+                y: data[1],
+              })),
+              borderColor: '#00BB7F',
+              tension: 0.3,
+              pointBackgroundColor: 'white',
+              pointBorderColor: 'blue',
+              pointRadius: 4,
+              pointHoverRadius: 6,
+            },
+          ],
         },
-      },
-      legend: {
-        data: ['Floor Price', 'Sold NFT Price'],
-        itemGap: 15,
-        inactiveColor: '#',
-        textStyle: {
-          color: '#fff',
-        }
-      },
-      xAxis: {
-        type: 'time',
-        boundaryGap: false,
-        axisLabel: {
-          fontFamily: 'Fira Code',
-          color: '#fff',
-          hideOverlap: true,
-        },
-      },
-      yAxis: {
-        type: 'value',
-        axisLabel: {
-          formatter: '{value} KSM',
-          fontFamily: 'Fira Code',
-          color: '#fff',
-        },
-        splitLine: {
-          lineStyle: {
-            color: '#f8f8f8',
-            opacity: '.2'
-          }
-        },
-        nameTextStyle: {
-          color: '#fff',
-          fontFamily: 'Fira Code',
-        },
-      },
-      dataZoom: {
-        type: 'slider',
-        bottom: '2%',
-      },
-      series: [
-        {
-          name: 'Floor Price',
-          type: 'line',
-          smooth: 'true',
-          lineStyle: {
-            color: '#d32e79',
+        options: {
+          plugins: {
+            zoom: {
+              limits: {
+                x: { min: 0 },
+                y: { min: 0 },
+              },
+              pan: {
+                enabled: true,
+              },
+              zoom: {
+                wheel: {
+                  enabled: true,
+                },
+                pinch: {
+                  enabled: true,
+                },
+                mode: 'xy',
+                onZoomComplete({ chart }) {
+                  chart.update('none');
+                },
+              },
+            },
           },
-          data: this.priceData[0],
-        },
-        {
-          name: 'Sold NFT Price',
-          type: 'line',
-          smooth: 'true',
-          lineStyle: {
-            color: '#00BB7F',
+          scales: {
+            x: {
+              type: 'time',
+              time: {
+                unit: 'day',
+                stepSize: 7,
+              },
+              grid: {
+                drawOnChartArea: false,
+                drawTicks: false,
+                drawBorder: false,
+              },
+              ticks: {
+                callback: (value) => {
+                  return value;
+                },
+                major: {
+                  enabled: true,
+                },
+                maxRotation: 0,
+                minRotation: 0,
+                color: '#fff',
+              },
+            },
+            y: {
+              ticks: {
+                callback: (value) => {
```

Comment:
I would take the unit from the vuex `store`

Reply:
Hello, I'm not sure about what you are referring to as the unit from the vuex store. The y-axis of the chart will always be in KSM. I also can't find the unit to use for the ticks at y-axis from the vuex store.

Reply:
> The y-axis of the chart will always be in KSM

Only applies only for the Kusama. 

We will run KodaDot on [Basilisk](https://github.com/galacticcouncil/Basilisk-node) which has BSX token